lib/checks/keyboard/has-at-least-one-main-after.js

```diff
@@ -0,0 +1,8 @@
+var hasMain = false;
+for (var i = 0; i < results.length && !hasMain; i++) {
+	hasMain = results[i].data;
```

Comment:
This doesn't look like it would work. Basically, you're taking the last item on `results` and assigning that as the result for everything. That doesn't seem right. What was this supposed to do? Please comment the code, as these double loops are confusing.

Reply:
@WilcoFiers @jongund 

```
var hasMain = false;

//iterate through results from each document
//stops if any document contains a main landmark
for (var i = 0; i < results.length && !hasMain; i++) {
	hasMain = results[i].data;
}

//if any document contains a main landmark, set all documents to pass the check
//otherwise, fail all documents
//since this is a page level rule, all documents either pass or fail the requirement
for (var i = 0; i < results.length; i++) {
	results[i].result = hasMain;
}

return results;
```

Do these comments clear up the behavior of the function?

Since this is a page level rule all documents either fail or pass this rule.

We only need one main in any of the documents (e.g. main document or an iframe) to pass this rule, 

If some documents fail this rule (e.g. iframe), people will think they need to have a main landmark in every iframe.

Does this make sense?

Are we getting the logic right here, do we need to update the individual document results?